picard/options.py

```diff
@@ -173,6 +173,12 @@
 BoolOption('setting', 'filter_cover_by_size', False)
 IntOption('setting', 'cover_minimum_width', 250)
 IntOption('setting', 'cover_minimum_height', 250)
+BoolOption('setting', 'resize_images_saved_to_tags', False)
+IntOption('setting', 'cover_tags_maximum_width', 1000)
```

Comment:
Those numbers should perhaps be stored in https://github.com/metabrainz/picard/blob/master/picard/const/defaults.py :

```python
DEFAULT_COVER_MIN_SIZE = 250
DEFAULT_COVER_MAX_SIZE = 1000
```

```python
IntOption('setting', 'cover_minimum_width', DEFAULT_COVER_MIN_SIZE)
IntOption('setting', 'cover_minimum_height', DEFAULT_COVER_MIN_SIZE)
IntOption('setting', 'cover_tags_maximum_width', DEFAULT_COVER_MAX_SIZE)
IntOption('setting', 'cover_tags_maximum_height', DEFAULT_COVER_MAX_SIZE)
IntOption('setting', 'cover_file_maximum_width', DEFAULT_COVER_MAX_SIZE)
IntOption('setting', 'cover_file_maximum_height', DEFAULT_COVER_MAX_SIZE)
```